VsViewer/vs_calc/spt_vs_correlations.py

```diff
@@ -89,4 +133,150 @@ def effective_stress_brandenberg(
         return stress, sigma, tao, b0, b1, b2
 
 
-SPT_CORRELATIONS = {"brandenberg_2010": brandenberg_2010}
+def kwak_2015(spt: SPT):
+    """
+    Baseline SPT-Vs correlation developed by Kwak et al. (2015).
+
+    Parameters
+    ----------
+    spt : SPT
+        The SPT object to use for the correlation.
+
+    Returns
+    -------
+    vs : np.ndarray
+        The Vs values for the given SPT object.
+    vs_sd : np.ndarray
+        The standard deviation of the Vs values for the given SPT object.
+    depth_values : np.ndarray
+        The depth values for the Vs values.
+    eff_stress : np.ndarray
+        The effective stress values for the Vs values.
+    """
+    # Ensures N60 is calculated before trying to get Vs
+    N60 = spt.N60
+    vs = []
+    vs_sd = []
+    depth_values = []
+    eff_stress = []
+    for depth_idx, depth in enumerate(spt.depth):
+        true_d = (
+            depth + 0.3048
+        )  # Spt testing driven a pile 18 inches into the ground in 3 incremental steps. the
+        # number of blows is ignored and only consider the total of the second and third increments. We interests
+        # in the vertical effective stress after second increments hence add 12 inches(0.3 m) on top of the start
+        # depth given
+        cur_N60 = N60[depth_idx]
+        if cur_N60 > 0:
+            stress, sigma, tao, b0, b1, b2 = effective_stress_kwak(
+                true_d, spt.soil_type[depth_idx]
+            )
+            lnVs = (
+                b0 + b1 * np.log(cur_N60) + b2 * np.log(stress)
+            )  # (Kwak et al, 2015)
+            total_std = np.sqrt(tao**2 + sigma**2) #NOTE I HAVE NOT CODED THE CORRECT STDEV YET, THIS IS STILL BRANDENBERG
+            vs.append(np.exp(lnVs))
+            vs_sd.append(total_std)
+            depth_values.append(depth)
+            eff_stress.append(stress)
+    return np.asarray(vs), np.asarray(vs_sd), np.asarray(depth_values), np.asarray(eff_stress)
+
+
+def effective_stress_kwak(
+    depth: float, soiltype: SoilType = SoilType.Clay, water_table_depth: Optional[float] = 2
+):
+    """
+    Gets the effective stress for the given depth and soil type.
+
+    Parameters
+    ----------
+    depth : float
+        The depth to get the effective stress for.
+    soiltype : SoilType
+        The soil type to use for the effective stress calculation.
+    water_table_depth : float
+        The depth of the water table, default is 2 m below the ground surface.
```

Comment:
optional

Reply:
How about considering adding one of the SPT-Vs correlations from S. Rahimi et al. (2020), using Christchurch measurement data?

Reply:
And Is it possible to download the predicted Vs values for each section (not only for "Result")?

Reply:
> And Is it possible to download the predicted Vs values for each section (not only for "Result")?

Currently the download should be giving you a file each for each CPT/SPT with their correlation pairs with the estimated Vs and Vs_SD. Not sure if that's what your already asking for though.

Reply:
> How about considering adding one of the SPT-Vs correlations from S. Rahimi et al. (2020), using Christchurch measurement data?

More correlations can for sure be added if it would be useful for the team. If you are looking at using it then it would be good to add I reckon. This would be done in a different Pull Request however, but if you want to send through some details about the correlation then it can be looked at to implement and if there is any demo code for it (doesn't matter the state of it) that shows how it can be done then it can be integrated easier into this.